osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -18,169 +15,113 @@
-                            Masking = true,
+                            FillMode = FillMode.Fit,
                             Children = new[]
                             {
                                 hitExplosionContainer = new Container<HitExplosion>
                                 {
                                     RelativeSizeAxes = Axes.Both,
-                                    FillMode = FillMode.Fit,
                                     Blending = BlendingParameters.Additive,
                                 },
-                                HitTarget = new SkinnableDrawable(new TaikoSkinComponent(TaikoSkinComponents.HitTarget), _ => new HitTarget())
+                                HitTarget = new SkinnableDrawable(new TaikoSkinComponent(TaikoSkinComponents.HitTarget), _ => new TaikoHitTarget())
                                 {
-                                    Anchor = Anchor.CentreLeft,
-                                    Origin = Anchor.Centre,
                                     RelativeSizeAxes = Axes.Both,
-                                    FillMode = FillMode.Fit
                                 }
                             }
                         },
-                        barlineContainer = new ProxyContainer
-                        {
-                            RelativeSizeAxes = Axes.Both,
-                            Padding = new MarginPadding { Left = HIT_TARGET_OFFSET }
-                        },
-                        new Container
-                        {
-                            Name = "Hit objects",
-                            RelativeSizeAxes = Axes.Both,
-                            Padding = new MarginPadding { Left = HIT_TARGET_OFFSET },
-                            Masking = true,
-                            Child = HitObjectContainer
-                        },
-                        kiaiExplosionContainer = new Container<KiaiHitExplosion>
+                        hitTargetOffsetContent = new Container
                         {
-                            Name = "Kiai hit explosions",
                             RelativeSizeAxes = Axes.Both,
-                            FillMode = FillMode.Fit,
-                            Margin = new MarginPadding { Left = HIT_TARGET_OFFSET },
-                            Blending = BlendingParameters.Additive
-                        },
-                        judgementContainer = new JudgementContainer<DrawableTaikoJudgement>
-                        {
-                            Name = "Judgements",
-                            RelativeSizeAxes = Axes.Y,
-                            Margin = new MarginPadding { Left = HIT_TARGET_OFFSET },
-                            Blending = BlendingParameters.Additive
+                            Children = new Drawable[]
+                            {
+                                barlineContainer = new ProxyContainer
```

Comment:
These also shouldn't be masked as major barlines have triangles that exceed the bounds of the playfield.

Reply:
That explains why I couldn't see the triangles..

Reply:
Removing masking altogether seems to be fine. Do you know any reason masking was previously required?

Reply:
I don't remember, no :/